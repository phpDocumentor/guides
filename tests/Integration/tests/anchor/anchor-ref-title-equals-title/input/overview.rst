:orphan:


========
Overview
========

.. _rst-overview:

Overview 1
==========

RST Overview content

.. _sphinx-overview:

Overview 2
==========

Sphinx Overview content
