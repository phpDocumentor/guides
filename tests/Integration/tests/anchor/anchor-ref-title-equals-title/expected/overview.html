<!-- content start -->
    <div class="section" id="overview">
    <h1>Overview</h1>

            <div class="section" id="overview-1">
    <h2>Overview 1</h2>

            <a id="rst-overview"></a>
            <p>RST Overview content</p>
    </div>

            <div class="section" id="overview-2">
    <h2>Overview 2</h2>

            <a id="sphinx-overview"></a>
            <p>Sphinx Overview content</p>
    </div>

    </div>

<!-- content end -->
